### PR TITLE
PoojaIsaac/notifications app

### PR DESCRIPTION
**Pull request by idelgado2**
Merging notification prototype into development branch. I modified  navigation.component.ts, loginflag.service.ts, backend.service.ts, routes.js, and UserController.js

### CODE REVIEW

File: src/proxy.conf.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "/api/*": {
-    "target": "http://csse.usc.edu:3000",
```
Comment (frederickjjoubert) [inline src/proxy.conf.json]: I don't think this should be pushed to develop. Could you please fix this?

File: yarn.lock [hunk 2]
```diff
@@ -0,0 +1,7974 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```
Comment (frederickjjoubert) [inline yarn.lock]: I think this might need to be a part of the .gitignore

Comment (idelgado2) [reply yarn.lock]: I installed a package with yarn so we should keep this.

**Comment by frederickjjoubert** [comment]
Looks good to me!

**Pull request merged**